ConventionFileFolder/convention.md

```diff
@@ -0,0 +1,9 @@
+# Convention
+
+## Convention of the file name
+
+The file name should be in the camel case format. The file name should be in the format of `documentName.md`. For example, `convention.md`, `naming.md`, `fileNaming.md`, etc.
+
+## Convention of the folder name
+
+The folder name should be in the pascal case format. The folder name should be in the format of `FolderName`. For example, `Document`, `Convention`, `Naming`, etc.
```

Comment:
You've given 3 examples that don't explain how it works with a compound name like "myfolder". Add an example with a compound noun so that we understand that both words need to be capitalised.